### ISSUE TITLE
Messages may be broken across chunks inside stream and fail parsing

### ISSUE THREAD

**Issue by ccmcintyre-kingsds** [Kings-Distributed-Systems/Bifrost#45]
The stream used to communicate JS scripts to the Node.js process can, under some conditions (heavy-load multi-user environments?), require multiple message chunks to transfer a single script. At present the _transform function of the stream in main.js assumed that scripts are received as single atomic chunks, and attempts to parse them as JSON objects. Long scripts transfered under demanding system conditions (or any other message broken into more than one chunk, for whatever reason) will fail to parse and cause an error to be raised in the stream.

**Comment by ccmcintyre-kingsds** [comment]
Simple repeatable case:
```
import random
from bifrost import node

flood_length = 65504
flood_string = "let floodString='"
for i in range(flood_length):
  flood_string = flood_string + str(random.randint(0,9))
flood_string = flood_string + "';"

node.run(flood_string)
```

The above example, with the given configuration and after some initial parsing and padding, leads to a script of exactly length 65537 by the time Bifrost calls `process.stdin.write` on it. On my dev machine, this length consistently fails, and any shorter length consistently passes. This is, it turns out, 2^16+1. I would not be surprised if the default chunk length under trivial load is related to that value.

**Comment by ccmcintyre-kingsds** [comment]
Related to previous issue #11 
Resolved by pull request #51 
Enhanced by pull request #52
